contributing/building-a-bento-amp-extension.md

```diff
@@ -670,8 +670,9 @@ $ amp check-types
     -   [amp-timeago](https://github.com/ampproject/amphtml/pull/26507)
     -   [amp-base-carousel](https://github.com/ampproject/amphtml/pull/29303)
     -   [amp-stream-gallery](https://github.com/ampproject/amphtml/pull/30597)
--   Adding video components which use `VideoWrapper`
+-   Adding video components which use `VideoWrapper`. [You may also follow the guide to Building a Bento Video Component.](./building-a-bento-video-player.md)
```

Comment:
Question for my understanding: Do relative links like this work if this page gets pulled into amp.dev?

Reply:
From results for `(./` in `extensions/**/amp-*.md`, it looks like they do work.